functests/test_resources/dynamic_tolerance_story.yaml

```diff
@@ -1,153 +1,157 @@
+---
+# Demonstration of applying a Dynamic Tolerance compliance policy to an asset that undergoes
+# events that may or may not make the asset compliant or non-compliant.
+#
+# The operation field is a string that represents the method bound to an endpoint and
+# the args and kwargs correspond to the arguments to such a method.
+#
+# NB the assets and events endpoints require all values to be strings. Other values may
+# be of the correct type such as confirm which is a boolean.
+#
+# For example kwargs.attrs.ev_pump is a string - internally archivist will see this
+# as a boolean.
 operations:
-
-  - operation: CREATE_ASSET
+  - operation: assets.create
     to_print: Create new EV Pump with id 1.
-    args:
+    kwargs:
       attrs:
-        arc_description: ev pump 1
+        arc_display_name: ev pump 1
         ev_pump: "true"
       confirm: true
 
-
   # create some policies
-  - operation: CREATE_COMPLIANCE_POLICY_DYNAMIC_TOLERANCE
+  - operation: compliance_policies.create_from_data
     to_print: Create a compliance policy that checks an EV pump maintenance requests are serviced within a reasonable time frame.
-    policy_id: ev maintenance policy 1
```

Comment:
We've lost the friendly name reference to this policy. Don't we need that? eg for deleting it in clean-up at the end?

Reply:
I could not find it used anywhere so removed it...